src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/Azure.Configuration.csproj

```diff
@@ -17,6 +17,7 @@
     <NoWarn>$(NoWarn);3021</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <AssemblyName>Azure.ApplicationModel.Configuration</AssemblyName>
+    <EnableClientSdkAnalyzers>true</EnableClientSdkAnalyzers>
```

Comment:
Why flip the switch? Don't you want to have this on by default?

Reply:
This is included in a lot of track 1 projects that don't have complying namespaces and are not expected to.

This set of analyzers is track 2 specific so I've made it opt-in for now.

Reply:
I've defined an IsClientLibrary property (https://github.com/Azure/azure-sdk-for-net/blob/master/Directory.Build.props#L6) that can be used to distinguish track 1 and track 2 if you want to default it to true for just that set.

Reply:
👍 Exactly what I needed.